parts/k8s/containeraddons/kubernetesmasteraddons-aad-pod-identity-deployment.yaml

```diff
@@ -104,15 +104,15 @@ spec:
       hostNetwork: true
       containers:
       - name: nmi
-        image: "mcr.microsoft.com/k8s/aad-pod-identity/nmi:1.2"
```

Comment:
An addon in the `containeraddons` directory should make at least one container image user-configurable at a minimum, and resource configuration as well, if appropriate.